Documentation/user-guides/storage.md

```diff
@@ -28,16 +28,23 @@ It is recommended to use volumes that have high I/O throughput therefore we're u
 The `StorageClass` that was created can be specified in the `storage` section in the `Prometheus` resource.
 
 ```yaml
-apiVersion: "monitoring.coreos.com/v1alpha1"
-kind: "Prometheus"
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: Prometheus
 metadata:
-  name: "persisted"
+  name: persisted
 spec:
-  storage:
-    class: ssd
-    resources:
-      requests:
-        storage: 40Gi
+  replicas: 1
+  resources:
+  pvc:
+    metadata:
+      name: myclaim
+    spec:
+      accessModes:
+        - ReadWriteOnce
```

Comment:
`ReadWriteOnce` should be the only mode used with Prometheus, so we should always override this, therefore we can drop it in this example.